solutions/quickstart/provider.tf

```diff
@@ -0,0 +1,28 @@
+provider "ibm" {
+  ibmcloud_api_key      = var.ibmcloud_api_key
+  region                = var.region
+  visibility            = var.provider_visibility
+  private_endpoint_type = (var.provider_visibility == "private" && var.region == "ca-mon") ? "vpe" : null
+}
+
+provider "elasticsearch" {
+  alias                 = "ibm_es"
+  username              = module.icd_elasticsearch.service_credentials_object.credentials["elasticsearch_admin"].username
+  password              = module.icd_elasticsearch.service_credentials_object.credentials["elasticsearch_admin"].password
+  url                   = "https://${module.icd_elasticsearch.service_credentials_object.hostname}:${module.icd_elasticsearch.service_credentials_object.port}"
+  cacert_file           = base64decode(module.icd_elasticsearch.service_credentials_object.certificate)
+  elasticsearch_version = "8.12"
+
+  # Below is required to avoid HEAD healthcheck failed error. Reference to similar issue and suggestion - https://github.com/phillbaker/terraform-provider-elasticsearch/issues/352
+  healthcheck = false
+}
+
+provider "restapi" {
```

Comment:
what module requires restapi?

Reply:
`watsonx.ai` requires restapi, if not declared, like we usually do and let the referenced module take care of it, If not used, getting the below error, hence explicitly added the provider block.

<img width="1040" height="179" alt="image" src="https://github.com/user-attachments/assets/f28743f4-b1aa-400a-9c87-0747e232ec18" />